tests/test_crash_reports.py

```diff
@@ -20,7 +20,7 @@ class TestCrashReports:
     @pytest.mark.parametrize(('product'), [
         'Firefox',
         'Thunderbird',
-        'SeaMonkey',
+        pytest.mark.xfail(reason='bug 1273175')('SeaMonkey'),
```

Comment:
Wow--I didn't even know you could compile an xfail with a parametrize. That's AMAZINGS!